src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -106,7 +110,7 @@ export default function DownloadCartPage() {
                 <div className="container">
                   <div className="subSectionContainer">
                     <span className="subSectionTitle">Complete Your Download</span>
-                    <DownloadListStats />
+                    <DownloadListStats numBytes={data.sumOfFileSizesAvailableForDownload} numFiles={data.numberOfFilesAvailableForDownload}/>
```

Comment:
This was the cyclical loading loop that showed up when integrating into SWC.  The child component also asked for the Download List stats and when loaded it caused the child to be re-mounted and start the query again.  Result: Accidental DOS attack on Synapse.